On web, allow recreating event loop when using `spawn` (#2720)
rust-windowing/winit: CHANGELOG.md

# Unreleased

> - On web, allow event loops to be recreated with `spawn`.

Reviewer: ```suggestion
- On Web, allow `EventLoop` to be recreated after using `spawn`.
```
